views/list-job-roles.html

```diff
@@ -24,13 +24,15 @@ <h2>Job Roles</h2>
             <tr>
                 <th scope="col">Job Title</th>
                 <th scope="col">Capability</th>
+                <th scope="col">Training</th>
             </tr>
             </thead>
             <tbody>
             {% for jobrole in jobroles %}
             <tr>
                 <td> {{jobrole.roleTitle}}</td>
                 <td> {{jobrole.capabilityName}}</td>
+                <td><a href="http://localhost:3000/view-training/{{jobrole.bandId}}">View Courses</a></td>
```

Comment:
How could we write this so that we don't need to include localhost? Would something like this work?

```suggestion
                <td><a href="/view-training/{{jobrole.bandId}}">View Courses</a></td>
```